tests/utils/test_types.py

```diff
@@ -14,12 +14,12 @@ def test_get_type(self):
         If we pass in an optional type, it should return the non-optional type.
         """
         # Should return the underlying type, as they're all optional:
-        self.assertIs(get_type(t.Optional[str]), str)
-        self.assertIs(get_type(t.Optional[t.List[str]]), t.List[str])
-        self.assertIs(get_type(t.Union[str, None]), str)
+        self.assertIs(get_type(Optional[str]), str)
+        self.assertIs(get_type(Optional[list[str]]).__mro__, list[str].__mro__)
+        self.assertIs(get_type(Union[str, None]), str)
 
         # Should be returned as is, because it's not optional:
-        self.assertIs(get_type(t.List[str]), t.List[str])
+        self.assertIs(get_type(list[str]).__mro__, list[str].__mro__)
```

Comment:
I changed this slightly. Comparing `__mro__` does work, but I was confused by it at first. I changed the assertions to `assertEqual` and this works instead.